Makefile

```diff
@@ -53,21 +54,22 @@ check-licenses: fetch
 		"$(BUILDER_IMAGE)" \
 		bash -c "cargo deny --all-features check --disable-fetch licenses bans sources"
 
-# Builds, Lints and Tests the Rust workspace
-build: fetch check-licenses
+# Builds, Lints, and Tests the Rust workspace locally
+build: check-licenses
 	cargo fmt -- --check
-	cargo build --locked
 	cargo test --locked
+	cargo build --locked
 
-brupop-image: fetch check-licenses
+# Builds only the brupop image. Useful target for CI/CD, releasing, etc.
+brupop-image:
 	docker build $(DOCKER_BUILD_FLAGS) \
 		--build-arg UNAME_ARCH="$(UNAME_ARCH)" \
 		--build-arg BUILDER_IMAGE="$(BUILDER_IMAGE)" \
 		--tag "$(IMAGE_NAME)" \
 		--network none \
 		-f Dockerfile .
 
-dist: brupop-image
+dist: check-licenses brupop-image
```

Comment:
This target should utilize the check-license / fetch path since it's creating a distribution (and we should ensure we have checked the licenses locally)